margaret254/SendIt
# [finishes #161998440] Modifies and fixes some bugs.

#### What does this PR do?
Fixes bugs and makes some changes in ll the other pages.
#### Description of Task to be completed?
Validate the sign up and login buttons, such that when the user clicks on the buttons, the user is taken to the views page.
#### How should this be manually tested?
open the following URL: to see how the pages are after the fix:
clone the repository at https://github.com/margaret254/SendIt.git. open the /UI directory and then open the signup.html file at your browser.
#### What are the relevant pivotal tracker stories?
#161998440
